src/mpags-cipher.cpp

```diff
@@ -1,145 +1,100 @@
-#include "ProcessCommandLine.hpp"
-#include "RunCaesarCipher.hpp"
-#include "TransformChar.hpp"
-
+//Main function
 #include <cctype>
-#include <fstream>
 #include <iostream>
 #include <string>
 #include <vector>
+#include <fstream>
```

Comment:
It is a common convention to have the includes ordered alphabetically and to separate external and internal (e.g. TransformChar) includes by an empty line